modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts

```diff
@@ -1077,3 +1079,182 @@ test(
+			const user = await apiHelpers.headlessAdminUser.postUserAccount();
+			userAccounts.push(user);
+		}
+
+		const organization =
+			await apiHelpers.headlessAdminUser.postOrganization();
+
+		await usersAndOrganizationsPage.goToOrganizations();
+
+		await (
+			await usersAndOrganizationsPage.organizationActionsMenu(
+				organization.name
+			)
+		).click();
+
+		await usersAndOrganizationsPage.assignUsersMenuItem.click();
+
+		for (const user of userAccounts) {
+			await (
+				await usersAndOrganizationsPage.assignUsersCheckbox(user.name)
+			).check();
+		}
+
+		await usersAndOrganizationsPage.assignUsersDoneButton.click();
+
+		for (const user of userAccounts) {
+			apiHelpers.data.push({
+				id: `${organization.id}_${user.emailAddress}`,
+				type: 'organizationUserAccountAssociation',
+			});
+		}
+
+		await waitForAlert(page);
+
+		for (const user of userAccounts) {
+			await expect(
+				(
+					await usersAndOrganizationsPage.organizationUsersTableRow(
+						1,
+						user.name,
+						true
+					)
+				).row
+			).toBeVisible();
+		}
+	}
+);
+
+test(
+	'Can change user password',
+	{tag: '@LPD-50958'},
+	async ({accountSettingsPage, apiHelpers, page}) => {
+		const user = await apiHelpers.headlessAdminUser.postUserAccount();
+
+		userData[user.alternateName] = {
+			name: user.givenName,
+			password: 'test',
+			surname: user.familyName,
+		};
+
+		await performLogout(page);
+		await performLoginViaApi(page, user.alternateName);
+
+		await accountSettingsPage.goToAccountSettings();
+		await accountSettingsPage.passwordMenuItem.click();
+		await accountSettingsPage.currentPasswordInput.fill('test');
+
+		const newPassword = getRandomString();
+
+		await accountSettingsPage.newPasswordInput.fill(newPassword);
+		await accountSettingsPage.reenterPasswordInput.fill(newPassword);
+		await accountSettingsPage.saveButton.click();
+
+		await waitForAlert(page);
+
+		userData[user.alternateName] = {
+			name: user.givenName,
+			password: newPassword,
+			surname: user.familyName,
+		};
+
+		await performLogout(page);
+		await performLoginViaApi(page, user.alternateName);
+
+		await expect(accountSettingsPage.userPersonalMenuButton).toBeVisible();
+	}
+);
+
+test(
+	'Change user password invalid',
+	{tag: '@LPD-50958'},
+	async ({accountSettingsPage, apiHelpers, page}) => {
+		const user = await apiHelpers.headlessAdminUser.postUserAccount();
+
+		userData[user.alternateName] = {
+			name: user.givenName,
+			password: 'test',
+			surname: user.familyName,
+		};
+
+		await performLogout(page);
+		await performLoginViaApi(page, user.alternateName);
+
+		await accountSettingsPage.goToAccountSettings();
+		await accountSettingsPage.passwordMenuItem.click();
+
+		await accountSettingsPage.newPasswordInput.fill('password');
+		await accountSettingsPage.reenterPasswordInput.fill('password');
+		await accountSettingsPage.saveButton.click();
+
+		await expect(
+			page.getByText('The Current Password field is required.')
+		).toBeVisible();
+
```

Comment:
can you add also the case (in this test) where you fill the current password with a random value (so different from the real current password)?